tests/frontend/test_tdmprogram.py

```diff
@@ -18,11 +18,13 @@
 from strawberryfields import ops
 from strawberryfields.tdm import tdmprogram
 
+pytestmark = pytest.mark.frontend
+
 # make test deterministic
 np.random.seed(42)
 
 
-def singleloop(r, alpha, phi, theta, copies, shift="default", hbar=2):
```

Comment:
Not entirely sure about the reason here (recall hearing about the change, but can't recall the exact reason :thinking: )